oak/server/oak_node.h

```diff
@@ -17,40 +17,45 @@
 #ifndef OAK_SERVER_OAK_NODE_H_
 #define OAK_SERVER_OAK_NODE_H_
 
+#include <unordered_map>
+
 #include "absl/base/thread_annotations.h"
-#include "absl/synchronization/mutex.h"
 #include "absl/types/span.h"
 #include "oak/proto/node.grpc.pb.h"
+#include "oak/server/channel.h"
 #include "src/interp/interp.h"
 
 namespace oak {
 
+using Handle = uint64_t;
+
+// Keep in sync with /rust/oak/src/lib.rs.
+const Handle LOGGING_CHANNEL_HANDLE = 1;
+const Handle GRPC_CHANNEL_HANDLE = 2;
+const Handle GRPC_METHOD_NAME_CHANNEL_HANDLE = 3;
```

Comment:
It feels slightly odd that two of the three standard channels are not really bidirectional, but I guess they're special cases and most everything else will be.

Reply:
Yeah I also don't like the asymmetry, but also this hardcoded table of channels will probably be replaced with something else, so let's keep talking about it.